build.gradle

```diff
@@ -53,6 +57,46 @@ buildscript {
         return audioSwitchBintrayPassword;
     }
 
+    ext.getAudioSwitchJfrogOssUsername = {
+        def audioSwitchJfrogOssUsername  = System.getenv("AUDIOSWITCH_JFROG_OSS_USERNAME");
+
+        if (audioSwitchJfrogOssUsername == null) {
+            logger.log(LogLevel.INFO, "Could not locate AUDIOSWITCH_JFROG_OSS_USERNAME environment variable. " +
+                    "Trying local.properties")
+            Properties properties = new Properties()
+            if (project.rootProject.file('local.properties').exists()) {
+                properties.load(project.rootProject.file('local.properties').newDataInputStream())
+                audioSwitchJfrogOssUsername = properties.getProperty('AUDIOSWITCH_JFROG_OSS_USERNAME')
+            }
+        }
+
+        if (audioSwitchJfrogOssUsername == null) {
+            logger.log(LogLevel.WARN, "AudioSwitch Jfrog OSS username unavailable.")
+        }
+
+        return audioSwitchJfrogOssUsername;
+    }
+
+    ext.getAudioSwitchJfrogOssPassword = {
```

Comment:
Could we use a common method to share this logic to check for an environment variable/property? I noticed this logic is duplicated in a few different areas like in `getAudioSwitchJfrogOssUsername()`, `getAudioSwitchBintrayUsername`, and `getAudioSwitchBintrayPassword`.

Reply:
Done

Reply:
Awesome! Thanks 👍